src/tests/Microsoft.PowerFx.Interpreter.Tests/DisplayNameTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Text;
+using Microsoft.PowerFx.Core.Tests;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class DisplayNameTests
+    {
+        [Fact]
+        public void DisplayNameTest_DropColumns()
+        {
+            var pfxConfig = new PowerFxConfig(Features.SupportColumnNamesAsIdentifiers);
+            var engine = new RecalcEngine(pfxConfig);
+
+            var recordType = RecordType.Empty()
+                .Add(new NamedFormulaType("logicalA", FormulaType.Number, displayName: "displayName"))
+                .Add(new NamedFormulaType("logicalB", FormulaType.Number));
+
+            var rv1 = RecordValue.NewRecordFromFields(
+                new NamedValue("logicalA", FormulaValue.New(1)),
+                new NamedValue("logicalB", FormulaValue.New(4)));
+            var rv2 = RecordValue.NewRecordFromFields(
+                new NamedValue("logicalA", FormulaValue.New(2)),
+                new NamedValue("logicalB", FormulaValue.New(5)));
+            var rv3 = RecordValue.NewRecordFromFields(
+                new NamedValue("logicalA", FormulaValue.New(3)),
+                new NamedValue("logicalB", FormulaValue.New(6)));
+
+            var tv = TableValue.NewTable(recordType, rv1, rv2, rv3);
+
+            var parameters = RecordValue.NewRecordFromFields(
+                new NamedValue("myTable", tv));
+
+            var result = engine.Eval("DropColumns(myTable, displayName)", parameters);                      
+            var output = result.ToExpression();
+
+            Assert.Equal("Table({logicalB:4},{logicalB:5},{logicalB:6})", output);
+
+            var displayExpression = engine.GetDisplayExpression("DropColumns(myTable, displayName)", parameters.Type);
+            var invariantExpression = engine.GetInvariantExpression("DropColumns(myTable, displayName)", parameters.Type);
+
+            Assert.Equal("DropColumns(myTable, displayName)", displayExpression);
+            Assert.Equal("DropColumns(myTable, logicalA)", invariantExpression);
```

Comment:
>DropColumns [](http://example.com/codeflow?start=26&length=11)

Not related to your change but 
ShowColumns is very interesting function and we should fix it.

Today following expression returns a type *[createdby: s]
Showcolumns(Accounts, “createdby”) 

After your change same expression customers will write as 
ShowColumns(Accounts, ‘Created By’)
But the return type still contains logical field name.
*[createdby: s]

This would be even more confusing after this change.

But DropColumns actually returns a type with display name in it.

Reply:
I'll test the return types with display names